src/fontra/views/editor/panel-selection-info.js

```diff
@@ -160,6 +164,21 @@ export default class SelectionInfoPanel extends Panel {
 
     const formContents = [];
     if (glyphName) {
+      formContents.push({
+        key: "glyphLocking",
+        type: "header",
+        label: "Glyph info",
+        auxiliaryElement: html.createDomElement("icon-button", {
+          "id": "glyphLocking",
+          "style": `width: 1.3em;`,
+          "src": glyphLocked
+            ? "/tabler-icons/lock.svg"
+            : "/tabler-icons/lock-open-2.svg",
+          "onclick": (event) => this._glyphLocking(varGlyphController.glyph),
```

Comment:
Suggest to call this method `_toggleGlyphLock`

Reply:
Done.